### PR TITLE
0019 - Create FQCN Autowiring

### PR DESCRIPTION
**Pull request by FabienPapet**
This PR follows discussion started in https://github.com/PrestaShop/ADR/issues/28

Votes

## Vote

Maintainer | Vote (✅  or ❌)
---------- | ----
@atomiix  | ✅
@eternoendless | ✅ (https://github.com/PrestaShop/ADR/issues/28#issuecomment-1295108011)
@jolelievre | ✅
@kpodemski | ✅
@matks |  ✅ 
@matthieu-rolland | ✅ (https://github.com/PrestaShop/ADR/issues/28#issuecomment-1307246591)
@NeOMakinG | 
@PululuK | 
@sowbiba | ✅️

### CODE REVIEW

**Review by atomiix — APPROVED** [review]

**Review by nicosomb — APPROVED** [review]

**Review by mflasquin — APPROVED** [review]

**Review by yanmakouf — APPROVED** [review]

**Review by jolelievre — APPROVED** [review]

**Comment by NeOMakinG** [comment]
Blank one, trusting my mates on this

**Review by kpodemski — APPROVED** [review]

**Comment by jolelievre** [comment]
Since 2 / 3 of the maintainers approved this ADR it is now accepted.

File: 0019-FQCN-Autowiring.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+# Context
```
Comment (jolelievre) [inline 0019-FQCN-Autowiring.md:1]: Hi @FabienPapet 

can you please add a title the date and the status accepted at the top of this file please?
Similarly to previous ADRs:
- https://github.com/PrestaShop/ADR/blob/master/0017-backward-compatibility-promise.md
- https://github.com/PrestaShop/ADR/blob/master/0018-horizontal-migration.md

**Review by sowbiba — APPROVED** [review]

**Review by matks — APPROVED** [review]

**Comment by matks** [comment]
Thank you everyone

**Pull request merged**